commit 2b1649a83afc917b66731a4ccaec64eca5f9861d
Author: Tim Harvey <tharvey@gateworks.com>
Date:   Tue Mar 30 08:19:44 2021 -0700

    arm64: dts: imx: Add i.mx8mm Gateworks gw7901 dts support
    
    The Gateworks GW7901 is an ARM based single board computer (SBC)
    featuring:
     - i.MX8M Mini SoC
     - LPDDR4 DRAM
     - eMMC FLASH
     - SPI FRAM
     - Gateworks System Controller (GSC)
     - Atmel ATECC Crypto Authentication
     - USB 2.0
     - Microchip GbE Switch
     - Multiple multi-protocol RS232/RS485/RS422 Serial ports
     - onboard 802.11ac WiFi / BT
     - microSD socket
     - miniPCIe socket with PCIe, USB 2.0 and dual SIM sockets
     - Wide range DC power input
     - 802.3at PoE
    
    Signed-off-by: Tim Harvey <tharvey@gateworks.com>
    Reviewed-by: Fabio Estevam <festevam@gmail.com>
    Signed-off-by: Shawn Guo <shawnguo@kernel.org>

diff --git a/arch/arm64/boot/dts/freescale/Makefile b/arch/arm64/boot/dts/freescale/Makefile
index 44890d56c194..25806c4924cb 100644
--- a/arch/arm64/boot/dts/freescale/Makefile
+++ b/arch/arm64/boot/dts/freescale/Makefile
@@ -41,6 +41,7 @@ dtb-$(CONFIG_ARCH_MXC) += imx8mm-var-som-symphony.dtb
 dtb-$(CONFIG_ARCH_MXC) += imx8mm-venice-gw71xx-0x.dtb
 dtb-$(CONFIG_ARCH_MXC) += imx8mm-venice-gw72xx-0x.dtb
 dtb-$(CONFIG_ARCH_MXC) += imx8mm-venice-gw73xx-0x.dtb
+dtb-$(CONFIG_ARCH_MXC) += imx8mm-venice-gw7901.dtb
 dtb-$(CONFIG_ARCH_MXC) += imx8mn-beacon-kit.dtb
 dtb-$(CONFIG_ARCH_MXC) += imx8mn-evk.dtb
 dtb-$(CONFIG_ARCH_MXC) += imx8mn-ddr4-evk.dtb